src/Adapter/CustomerService/CommandHandler/ForwardCustomerThreadHandler.php

```diff
@@ -306,7 +306,7 @@ private function createCustomerMessage(ForwardCustomerThreadCommand $command)
         $customerMessage = new CustomerMessage();
         $customerMessage->id_employee = (int) $this->context->employee->id;
         $customerMessage->id_customer_thread = (int) $command->getCustomerThreadId()->getValue();
-        $customerMessage->ip_address = (int) ip2long(Tools::getRemoteAddr());
+        $customerMessage->ip_address = (string) (int) ip2long(Tools::getRemoteAddr());
```

Comment:
ip2long returns an int so I think we can get rid of the first cast?
```suggestion
        $customerMessage->ip_address = (string) ip2long(Tools::getRemoteAddr());
```

Reply:
No !

`echo ((string) (int) false);` returns `0`
`echo ((string) false);` returns ``